package.json

```diff
@@ -15,7 +15,8 @@
     "jest": "^23.1.0",
     "jest-runner-eslint": "^0.5.0",
     "prettier": "^1.12.1",
-    "serverless": "^1.35.0"
+    "serverless": "^1.35.0",
+    "aws-sam-local": "^0.2.11"
```

Comment:
can we add `packages/appsync-emulator-serverless/__test__/example/__pycache__/` to .gitignore ?

Reply:
and `packages/appsync-emulator-serverless/__test__/example/gocomposedjson/gocomposedjson` ?